docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst

```diff
@@ -244,6 +244,31 @@ The coverage propensity for wasting treatment parameter :math:`C` for any given
 
   This strategy assumes that simulants who are treated for MAM and SAM once are no more likely to be treated again than simulants who have never been treated for SAM or MAM (despite need).
 
+Targeted Coverage for MAM Treatment
++++++++++++++++++++++++++++++++++++
+
+In accordance with `new guidelines from the WHO <https://files.magicapp.org/guideline/a3fe934f-6516-460d-902f-e1c7bbcec034/published_guideline_7330-1_1.pdf>`_ on the prevention and management of acute 
```

Comment:
is magicapp.org really the definitive domain to get these guidelines?  that is so spammy!

Reply:
Okay I was curious and it appears yes. In other press releases they link to magicapp as well so I think this is the "best" link.